dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceQueryParams.java

```diff
@@ -991,12 +991,12 @@ public enum OrderColumn {
     // TODO(tracker): remove with old tracker
     CREATED("created", CREATED_ID, MAIN_QUERY_ALIAS),
     CREATED_AT("createdAt", CREATED_ID, MAIN_QUERY_ALIAS),
-    CREATED_AT_CLIENT("createdAtClient", "createdatclient", MAIN_QUERY_ALIAS),
+    CREATED_AT_CLIENT("createdAtClient", "createdAtClient", MAIN_QUERY_ALIAS),
```

Comment:
Aren't the column in the DB all lowercase?

Reply:
yeah, that was not meant to be part of the PR